blogs/2016/08/19/goodbyeuservoice.md

```diff
@@ -0,0 +1,94 @@
+---
+Order: 14
+TOCTitle: Goodbye User Voice
+PageTitle: Goodbye User Voice
+MetaDescription: VS Code is closing User Voice in favor of Github reactions.  
+Date: 2016-08-18
+ShortDescription: VS Code is closing User Voice in favor of Github reactions. 
+Author: Wade Anderson
+---
+
+# Goodbye User Voice, hello Github Reactions! 
+
+August 15, 2016 by Wade Anderson, [@waderyan_](https://twitter.com/waderyan_)
+  
+## Feedback Channels
+
+VS Code's [User Voice site](https://visualstudio.uservoice.com/forums/293070-visual-studio-code) has always been the place for the community to suggest and vote on feature requests for VS Code. The ability to see the requests with the top votes or the most activity has allowed us to prioritize the Code backlog and deliver many of the top requests over the past 18 months. For example, we’ve added [Extensions](https://visualstudio.uservoice.com/forums/293070-visual-studio-code/suggestions/9181439-plugins), [Tabs](https://visualstudio.uservoice.com/forums/293070-visual-studio-code/suggestions/7752519-implement-tabs) and an [integrated terminal](https://visualstudio.uservoice.com/forums/293070-visual-studio-code/suggestions/7752357-integrated-terminal), all because of your feedback and participation in User Voice. 
```

Comment:
Use "Visual Studio Code" for the first reference in a doc, then "VS Code" after that.